eva-accession-ws/pom.xml

```diff
@@ -59,6 +59,17 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>2.6</version>
+                <configuration>
+                    <delimiters>
+                        <!-- Provide a delimiter that doesn't clash with Spring ${} placeholders nor @ in URLs -->
+                        <delimiter>£</delimiter>
```

Comment:
I suggest to replace this with a character that can be easily typed with a non-UK keyboard.